### PR TITLE
Fix lovelace

### PR DESCRIPTION
**Pull request by quthla**
Makes Lovelace overview when Lovelace selected as default
Fixes cursor being text for dropdown in FF

### CODE REVIEW

File: src/components/ha-sidebar.js [hunk 1]
```diff
@@ -226,6 +226,10 @@ class HaSidebar extends
       checks--;
     } while (checks > 0 && target !== null && !attr);
 
+    if (attr === 'states') {
```
Comment (balloob) [inline src/components/ha-sidebar.js]: This is not right. The sidebar shouldn't know about local storage.

**Review by balloob — CHANGES_REQUESTED** [review]

File: src/components/ha-sidebar.js (hunk 1 above)
Comment (quthla) [reply src/components/ha-sidebar.js]: Then what do you suggest

File: src/layouts/home-assistant-main.js [hunk 2]
```diff
@@ -126,6 +126,10 @@ class HomeAssistantMain extends NavigateMixin(EventsMixin(PolymerElement)) {
   _computeDisableSwipe(hass) {
     return NON_SWIPABLE_PANELS.indexOf(hass.panelUrl) !== -1;
   }
+
+  defaultPage() {
```
Comment (balloob) [inline src/layouts/home-assistant-main.js]: Should be a property and it's internal so prefix name with underscore

Comment (quthla) [reply src/layouts/home-assistant-main.js]: As there are only 2 methods in that class with an underscore, can the rest be considered not internal then?

Comment (balloob) [reply src/layouts/home-assistant-main.js]: No, probably needs fixing (but in a different PR)

Comment (quthla) [reply src/layouts/home-assistant-main.js]: Question is if the underscore here makes sense at all as none of the methods, including connectedCallback etc, is ever publicly accessed.

Comment (balloob) [reply src/layouts/home-assistant-main.js]: connectedCallback is a public API as defined by custom elements. It is called by the browser.

**Review by balloob — APPROVED** [review]

**Pull request merged**